.github/workflows/ci.yml

```
@@ -0,0 +1,13 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Can we have this run on all branches? I see no harm in "over-running" the github actions

```suggestion
  push:
    branches:
      - '**'
  pull_request:
    branches:
      - '**'
```

---

yeah we can but it was good to not burn time or resources for every push. I am ok if you want to enable it.

---

@spirosoik How does this work on GitHub actions, I mean the cost of CI runs/time/resources? These plugin repos are not as active as e.g. server and webapp, so I don't think we'll be over-using the CI resources

---

https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions

---

feel free to raise a PR in the actions workflows accordingly.